extensions/smallrye-metrics/runtime/src/main/java/io/quarkus/smallrye/metrics/runtime/MicrometerGCMetrics.java

```diff
@@ -147,28 +148,34 @@ public void startWatchingNotifications() {
                     gcPauseMaxValue.set(duration); // update the maximum GC length if needed
                 }
                 if (!registry.getGauges().containsKey(pauseSecondsMaxMetricID)) {
-                    registry.register(new ExtendedMetadata(metricName + ".seconds.max",
-                            MetricType.GAUGE,
-                            MetricUnits.NONE,
-                            "Time spent in GC pause",
-                            true),
+                    registry.register(new ExtendedMetadataBuilder()
+                            .withName(metricName + ".seconds.max")
+                            .withType(MetricType.GAUGE)
+                            .withUnit(MetricUnits.NONE)
+                            .withDescription("Time spent in GC pause")
+                            .skipsScopeInOpenMetricsExportCompletely(true)
+                            .build(),
                             new LambdaGauge(() -> mapForStoringMax.get(causeAndAction).doubleValue() / 1000.0), tags);
                 }
 
-                ExtendedMetadata countMetadata = new ExtendedMetadata(metricName + ".seconds.count",
-                        MetricType.COUNTER,
-                        MetricUnits.NONE,
-                        "Time spent in GC pause",
-                        true,
-                        metricName.replace(".", "_") + "_seconds_count");
+                ExtendedMetadata countMetadata = new ExtendedMetadataBuilder()
+                        .withName(metricName + ".seconds.count")
+                        .withType(MetricType.COUNTER)
+                        .withUnit(MetricUnits.NONE)
+                        .withDescription("Time spent in GC pause")
```

Comment:
Is it normal that both the descriptions of this one and the one above are identical?

Reply:
I too find it a bit odd, but that's how Micrometer (in the latest version) does it, and we try to mimic it as closely as possible